verification/curator-service/ui/README.md

```diff
@@ -22,8 +22,24 @@ corresponding to this CSV format is available
 [here](https://docs.google.com/spreadsheets/d/1J-C7dq1rNNV8KdE1IZ-hUR6lsz7AdlvQhx6DWp36bjE).
 The following fields are currently incorporated:
 
+##### Case reference
 - `sourceEntryId`: Optional. The UUID provided by the data source to identify
 the specific case.
+
+##### Demographics
+- `gender`: Optional. One of `Female`, `Male`, `Non-binary/Third gender`, or
+`Other`.
+- `ageRangeStart`: Optional. Number (years) from 0 to 120.
+- `ageRangeEnd`: Optional. Number (years) from 0 to 120.
+- `ageRange`: Optional. Convenience field accepting an age range in the format
+`start-end`. If provided in this format, will be used in lieu of
+`ageRangeStart` and `ageRangeEnd`.
+- `ethnicity`: Optional.
+- `nationalities`: Optional. Delimit multiple nationalities with a semicolon
```

Comment:
should we link to the list of options in github? not necessarily in this PR

Reply:
Yeah -- for many of these fields, there should be that kind of linkage. @acansler is working on a more polished piece of documentation to link from the portal that should take this into account (and is the reason why I didn't go nuts with it in this PR).